content/en/docs/setup/independent/kubelet-integration.md

```diff
@@ -79,7 +79,7 @@ networking, or other host-specific parameters. The following list provides a few
   unless you are using a cloud provider. You can use the `--hostname-override` flag to override the
   default behavior if you need to specify a Node name different from the machine's hostname.
 
-- Currently, the kubelet cannot automatically detects the cgroup driver used by the CRI runtime,
```

Comment:
@brycethomsen you may need to `s/detects/detect` instead of removing `cannot`.
cc @neolit123 for further tech review.

Reply:
thanks for the ping @xiangpengzhao 
yes, ATM the typo here is just `s/detects/detect`.

@brycethomsen 
the kubelet still does not support automatic detection. or has this changed? if yes, please link a PR.

what can be done here is to change the wording to something like:
> Currently, the kubelet cannot automatically detect the cgroup driver used by the CRI runtime, but the value of `--cgroup-driver` must match the cgroup driver used by the CRI runtime to ensure the health of the kubelet. kubeadm can automatically set the `--cgroup-driver` flag when starting a new kubelet instance in the case of Docker, but not for other CRIs.